### ISSUE TITLE
Remove Conda offline hacks when fixed in Conda.

### ISSUE THREAD

**Issue by jmchilton** [galaxyproject/galaxy#3635]
It worked in 4.2 but was broken again in 4.3. 

Depends on upstream Conda being fixed.

xref #3632

**Comment by mvdbeek** [comment]
Is there an issue on the conda issue tracker? Not that I think they really triage their issues, but sometimes they do respond.

**Comment by jmchilton** [comment]
@mvdbeek Yes - sorry about that - xref https://github.com/conda/conda/issues/2035.

**Comment by mvdbeek** [comment]
Hmm, I don't think that's the same issue we're seeing here. I think they have disabled the `--offline` mode on purpose in 4.3, since it does work in 3.X to 4.2. No clue why and why that's not mentioned in the release notes.

**Comment by mvdbeek** [comment]
Just to track this here, @ericenns posted this on [pastebin](http://pastebin.com/gktRkhLe)
```
2017-02-10 11:01:37,513 DEBUG [galaxy.tools.deps] Find dependency samtools version None
2017-02-10 11:01:37,513 DEBUG [galaxy.tools.deps] Using dependency samtools version None of type conda
2017-02-10 11:01:37,513 DEBUG [galaxy.tools.deps.conda_util] Executing command: /home/ericenns/miniconda2/bin/conda list --name __samtools@_uv_ --export > /tmp/tmpHWlL_P/tmp/jobdepsxNrTQo373e7bbee36e032187809f77033895b78158b1575365caf0cada674b44d010b8/__samtools@_uv_
2017-02-10 11:01:37,862 DEBUG [galaxy.tools.deps.conda_util] Executing command: /home/ericenns/miniconda2/bin/conda create -y --unknown --offline --prefix /tmp/tmpHWlL_P/job_working_directory/000/1/conda-env --file /tmp/tmpHWlL_P/tmp/jobdepsxNrTQo373e7bbee36e032187809f77033895b78158b1575365caf0cada674b44d010b8/__samto
ols@_uv_ > /dev/null


CondaError: RuntimeError(u'EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/linux-64/libgcc-5.2.0-0.tar.bz2\nThis command is using a remote connection in offline mode.\n',)

CondaError: RuntimeError(u'EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/linux-64/libgcc-5.2.0-0.tar.bz2\nThis command is using a remote connection in offline mode.\n',)

CondaError: RuntimeError(u'EnforceUnusedAdapter called with url https://repo.continuum.io/pkgs/free/linux-64/libgcc-5.2.0-0.tar.bz2\nThis command is using a remote connection in offline mode.\n',)
```
It sounds a little bit like there could be a setting to turn off this EnforceUnusedAdapter thingy ...

**Comment by mvdbeek** [comment]
And that's the commit that introduced it:
https://github.com/conda/conda/commit/3db15e8c8f3f337086d2c79227561212e008e4b7

**Comment by natefoo** [comment]
Should be fixed as of 4.3.18: conda/conda#4798

**Comment by natefoo** [comment]
Also, if the de-hack could be applied to 17.05 that'd be great (but if not I can cherry-pick for Main).

**Comment by natefoo** [comment]
Fixed in #4701.
